config/xsl/loadreport/sections/web-vitals.xsl

```diff
@@ -0,0 +1,159 @@
+                                                <xsl:with-param name="value" select="lcp"/>
+                                            </xsl:call-template>
+                                        </td>
+                                        <td class="value number web-vital">
+                                            <xsl:call-template name="web-vital-cell">
+                                                <xsl:with-param name="value" select="cls"/>
+                                                <xsl:with-param name="unit" select="''"/>
+                                                <xsl:with-param name="format" select="'#,##0.000'"/>
+                                            </xsl:call-template>
+                                        </td>
+                                        <td class="value number web-vital">
+                                            <xsl:call-template name="web-vital-cell">
+                                                <xsl:with-param name="value" select="fid"/>
+                                            </xsl:call-template>
+                                        </td>
+                                        <td class="value number web-vital">
+                                            <xsl:call-template name="web-vital-cell">
+                                                <xsl:with-param name="value" select="inp"/>
+                                            </xsl:call-template>
+                                        </td>
+                                        <td class="value number web-vital">
+                                            <xsl:call-template name="web-vital-cell">
+                                                <xsl:with-param name="value" select="ttfb"/>
+                                            </xsl:call-template>
+                                        </td>
+                                    </tr>
+                                </xsl:for-each>
+                            </tbody>
+                        </xsl:when>
+                        <xsl:otherwise>
+                            <tbody class="table-nosort">
+                                <tr>
+                                    <td colspan="7" class="no-data">No data available</td>
+                                </tr>
+                            </tbody>
+                        </xsl:otherwise>
+                    </xsl:choose>
+                </table>
+            </div>
+        </div>
+    </xsl:template>
+
+    <xsl:template name="web-vital-cell">
+        <xsl:param name="value"/><!-- the web-vital value node, such as "fcp" -->
+        <xsl:param name="unit" select="'ms'"/><!-- the unit (can be empty) -->
+        <xsl:param name="format" select="'#,##0'"/><!-- the format string to use when formatting scores -->
+
+        <xsl:if test="$value">
+            <xsl:variable name="count" select="$value/goodCount + $value/improveCount + $value/poorCount"/>
```

Comment:
What's wrong with good old '+'?

Reply:
Nothing. However, expressions for summing up values using '+' operator become hard to read when using more than 2 operands; XPath's `sum` function is more explicit and takes a variable number of arguments (so more operands can be easily added just in case Google decides to change its "spec").